src/components/Layout.tsx

```diff
@@ -18,22 +18,23 @@ const Wrapper = styled.div`
 function Layout() {
   const [showCrisisAlert, setShowCrisisAlert] = useState(true);
   const location = useLocation();
-  console.log("location ", location.pathname);
   return (
     <Wrapper className="display-flex flex-column">
       <Header basic color="primary" role="banner">
         <Banner />
         <div className="display-flex flex-justify-center border-bottom border-base-lighter">
           <div className="padding-top-2 padding-bottom-1 height-auto">
             <Grid row className="flex-align-center ">
-              <a href="/" title="Home" aria-label="Home">
+              <a href="/" title="Home" aria-label="Home" data-testid="homeLink">
```

Comment:
if these changes were all for the e2e tests (adding data-testids) maybe better to keep them out of this PR?

Reply:
removed and added the changes to fix the depths issue on GuidedSearch